front/app/modules/commercial/content_builder/admin/containers/index.tsx

```diff
@@ -14,49 +14,57 @@ import ContentBuilderToolbox from '../components/ContentBuilderToolbox';
 import ContentBuilderSettings from '../components/ContentBuilderSettings';
 import Container from '../components/CraftComponents/Container';
 import RenderNode from '../components/RenderNode';
+import ContentBuilderTopBar from '../components/ContentBuilderTopBar';
 
 const Wrapper = styled.div`
   flex: 0 0 auto;
-  width: 210px;
+  width: 212px;
 `;
 
 const ContainerInner = styled.nav`
   flex: 0 0 auto;
-  width: 210px;
+  width: 212px;
```

Comment:
Why did we change this value? It's supposed to be consistent with the width of the other navbar :) Or maybe I'm missing something here?